tests/v2/validation/rbac/rbac_psa_test.go

```diff
@@ -51,10 +53,11 @@ func (rb *PSATestSuite) SetupSuite() {
 	client, err := rancher.NewClient("", testSession)
 	require.NoError(rb.T(), err)
 
+	rb.namespace = defaultNamespace
```

Comment:
since this defaultNamespace is present in rbac.go and is globally available, we can remove this from here and just reference as defaultNamespace in the test.